docs/api/verify-service-provider-with-matching-api.swagger.yml

```
@@ -223,20 +226,17 @@ components:
             - AUTHENTICATION_FAILED
           description: Returned when there was a failure to authenticate.
```

SAML status code here?

---

@chrisholmes could you just check this is the correct status code 🤔 

```suggestion
          description: Returned when there was a failure to authenticate. The SAML response from GOV.UK Verify Hub contains the `urn:oasis:names:tc:SAML:2.0:status:AuthnFailed` status code.
```